src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -75,21 +71,20 @@ public ResponseEntity findAllUsers(@Valid @RequestBody SearchValue str) {
     return ResponseEntity.ok().build();
   }
 
-  @JsonView(UserView.UserProfile.class)
   @GetMapping("/bylogin/{login}")
   public ResponseEntity<?> getUserByLogin(@PathVariable("login") String login) {
     return ResponseEntity.ok().body(userService.getUserByLogin(login));
   }
 
-  @PutMapping("/post/{id}")
-  public ResponseEntity<?> addToFavorites(@Valid @PathVariable("id") PostRxDto post, @Valid @RequestBody UserRxDto user) {
+  @PutMapping("/like/{id}")
+  public ResponseEntity<?> addToFavorites(@PathVariable("id") Post post, @RequestBody User user) {
     userService.addLikers(post, user);
     return ResponseEntity.status(201).build();
   }
 
-  @DeleteMapping("/post/{id}")
-  public ResponseEntity<?> deleteFromFavorites(@Valid @PathVariable("id") PostRxDto post,
-                                               @Valid @RequestBody UserRxDto user) {
+  @DeleteMapping("/like/{id}")
+  public ResponseEntity<?> deleteFromFavorites(@PathVariable("id") Post post,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_aqAJjEPbGfqJY0&open=AWWU5_aqAJjEPbGfqJY0)

[ISSUE_KEY]: AWWU5_aqAJjEPbGfqJY0